src/Microsoft.Framework.CodeGeneration.Templating/Compilation/MetadataReferencesProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Framework.CodeGeneration.Templating.Compilation
+{
+    public class MetadataReferencesProvider
```

Comment:
I'm confused here. Did you just change how getting app references works on all platforms?